src/app/models/project.ts

```diff
@@ -13,8 +13,8 @@ export class Project {
     name: string;
     description: string;
     visibility: ProjectVisibility = ProjectVisibility.Private;
-    city: City;
-    scenario: Scenario;
+    city: City = {} as City;
```

Comment:
What's the reason for this change? While this might be work, an empty object for City/Scenario is not valid (these types have required parameters) and you're forcing the type system to accept a value that isn't actually a City/Scenario, which could lead to issues down the road. There's probably a better way depending on what the issue you're trying to get around is.